src/animate_deformedshape.rst

```diff
@@ -36,13 +40,13 @@ Examples:
     
 ::
 
-   animate_deformedshape(Model="TwoSpan_Bridge", LoadCase="Dynamic_GM1", dt=0.01)
+   ani = animate_deformedshape(Model="TwoSpan_Bridge", LoadCase="Dynamic_GM1", dt=0.01)
    
 The above command animates the deformedshape of structure by reading data from `TwoSpan_Bridge_ODB` with a sub-folder `Dynamic_GM1` at dt=0.01.
 
 ::
 
-   animate_deformedshape(Model="TwoSpan_Bridge", LoadCase="Dynamic_GM1",  dt=0.01, tStart=10.0, tEnd=20.0, 
+   ani = animate_deformedshape(Model="TwoSpan_Bridge", LoadCase="Dynamic_GM1",  dt=0.01, tStart=10.0, tEnd=20.0, 
```

Comment:
How does this affect the output?

Reply:
Using Spyder and Qt, if I don't return the animation object, the animation doesn't run. Are you able to get a running animation without returning the object?

Reply:
I see. Great! No, I didn't see the animation object on Spyder.